creds.go

```diff
@@ -46,21 +49,19 @@ func (creds *Credentials) String() string {
 	)
 }
 
-// Client is the entry point into all the functionality in this package. It
-// contains authentication credentials, and a service endpoint, which are
-// required for all HTTP operations.
+// Client provides access to all taskcluster HTTP APIs for a given service, on
+// a given API version, of a given deployed cluster.
 type Client struct {
+	// Version of client API, e.g. "v1"
+	Version string
```

Comment:
Perhaps ApiVersion?

Reply:
I'll call it `APIVersion` so `golint` doesn't [complain](https://github.com/golang/lint/blob/32a87160691b3c96046c0c678fe57c5bef761456/lint.go#L703). Good idea!